### PR TITLE
Stabilize CoreCLR packages for 1.1.0 RTM

### PR DESCRIPTION
**Pull request by gkhanna79**
@ericstj @weshaggard PTAL

@RussKeldorph FYI

### CODE REVIEW

**Comment by ericstj** [comment]
LGTM

File: src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.pkgproj [hunk 1]
```diff
@@ -31,9 +31,6 @@
     <ProjectReference Include="win\Microsoft.NETCore.Runtime.CoreCLR.pkgproj">
       <Platform>x86</Platform>
     </ProjectReference>
-    <ProjectReference Include="win\Microsoft.NETCore.Runtime.CoreCLR.pkgproj">
```
Comment (weshaggard) [inline src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.pkgproj]: does this mean we aren't shipping an updated arm package? That would mean that this coreclr package is not going to work in UWP scenarios correct?

**Comment by gkhanna79** [comment]
@dotnet-bot test Linux ARM Emulator Cross Debug Build please

**Comment by gkhanna79** [comment]
@RussKeldorph Have you seen this failure before? It came in this PR in https://ci.dot.net/job/dotnet_coreclr/job/release_1.1.0/job/arm_emulator_cross_debug_ubuntu_prtest/122/consoleFull#2096352037c7af6a29-b465-404a-b249-90a3911e4354:

```
13:34:11 FAILED   - JIT/Regression/CLR-x86-JIT/V1-M11-Beta1/b37131/b37131/b37131.sh
13:34:11                BEGIN EXECUTION
13:34:11                /bindings/tmp/arm32_ci_temp/coreclr/Windows_NT.x64.Debug/Tests/coreoverlay/corerun b37131.exe
13:34:11                Expected: 100
13:34:11                Actual: 4
13:34:11                END EXECUTION - FAILED
```

**Comment by RussKeldorph** [comment]
I have not, though my first guess would be that it is a manifestation of #6298 that we haven't seen unless it reproduces reliably.

**Comment by RussKeldorph** [comment]
Sorry, intended to mention @gkhanna79.

**Comment by gkhanna79** [comment]
Merging stabilization PR.

**Pull request merged**